FAULT TYPE: OPENSTACK_THROW_EXCEPTION-VOLUME
TARGET COMPONENT: /nova/compute/manager.py
TARGET CLASS: ComputeManager
TARGET FUNCTION DEF: _delete_instance+2452;8
FAULT POINT: self._cleanup_volumes(context, instance, bdms, raise_exc=False, detach=False)
